src/components/install-scratch/install-scratch.jsx

```diff
@@ -0,0 +1,146 @@
+const PropTypes = require('prop-types');
+const FormattedMessage = require('react-intl').FormattedMessage;
+const React = require('react');
+
+const OS_ENUM = require('../../lib/os-enum.js');
+const {CHROME_APP_RELEASED} = require('../../lib/feature-flags.js');
+
+const {isDownloaded, isFromGooglePlay} = require('./install-util.js');
+
+const FlexRow = require('../../components/flex-row/flex-row.jsx');
+const Steps = require('../../components/steps/steps.jsx');
+const Step = require('../../components/steps/step.jsx');
+
+require('./install-scratch.scss');
+
+const InstallScratch = ({
+    currentOS
+}) => (
+    <div className="blue install-scratch">
+        <FlexRow className="inner column">
+            <h2 className="title">
+                {CHROME_APP_RELEASED ? (
+                    <FormattedMessage
+                        id="installScratch.appHeaderTitle"
+                        values={{operatingsystem: currentOS}}
+                    />
+                ) : (
+                    <React.Fragment>
+                        {isDownloaded(currentOS) && (
+                            <FormattedMessage id="installScratch.desktopHeaderTitle" />
+                        )}
+                        {isFromGooglePlay(currentOS) && (
```

Comment:
If the ChromeApp isn't released, then I don't think the `isFromGooglePlay` can ever be true.

Reply:
Ah good point.